templates/includes/banner.html

```diff
@@ -0,0 +1,9 @@
+<section class="p-strip--light is-shallow">
+  <div class="row">
```

Comment:
you can use `u-fixed-width` instead of row>col-12

Reply:
Thanks, didn't know!